data/thunderbirdceg.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "Gokulnath Panneerselvam",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software developer with Cloud knowledge looking for remote roles",
+  "avatar": "https://github.com/thunderbirdceg.png",
+  "links": [
+    {
+      "name": "Follow me on Linkedin",
```

Comment:
Unfortunately, this isn't a valid LinkedIn link. Add the URL to your LinkedIn profile in the `url` field.